src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
...if somebody wants to dig in this, the multiline case is inferred in https://github.com/eslint/eslint/blob/master/lib/rules/comma-dangle.js and flowtypes are dealt with in https://github.com/babel/babel-eslint/blob/master/index.js and elsewhere.